Makefile.global

```diff
@@ -113,6 +113,9 @@ clean:
 	find . -name \*.gcno -o -name \*.gcda | xargs rm -f
 	find . -name \*.lo -o -name \*.o | xargs rm -f
 	find . -name \*.la -o -name \*.a | xargs rm -f
+	find . -name \*.diff -o -name \*.diff | xargs rm -f
```

Comment:
The `-o` means "or", so putting the same thing on both sides doesn't make much sense ^^ Maybe `find . -name \*.diff -o -name \*.out -o -name \*.log | xargs rm -f`?

Reply:
Always teaching me, thanks :smile: